### ISSUE TITLE
Modal Accessibility

### ISSUE THREAD

**Issue by ninja511** [carbon-design-system/carbon#1134]
Moving from Carbon-components-angular repo: 
https://github.com/IBM/carbon-components-angular/issues/64

Detailed description
Describe in detail the issue you're having.
Looking at the modal dialog to help with accessibility issues. Looks it help us with our tabbing order and focus. However I noticed that when I do the zoom 200% test the content of the dialog are not visible.

Is this a feature request (new component, new icon), a bug, or a general issue?
Zoom accessibility issues.

Is this issue related to a specific component?
modal

What offering/product do you work on? Any pressing ship or release dates we should be aware of?
IBM Watson Recruitment, Accessibility deadline of Dec 10, 2018.

Steps to reproduce the issue
Using the StoryBook, zoom out to 225% (use ctrl ++).

**Comment by asudoh** [comment]
Hi @ninja511 thank you for reporting! Do you have a screenshot demonstrating the problem? Also, did you try carbondesignsystem.com site to see if it reproduces? Thanks!

**Comment by ninja511** [comment]
@aradhikaIbm opened the issue - please respond to above thanks.

**Comment by aradhikaIbm** [comment]
I actually i'm starting to play with more. Looks like maybe zoom is ok. But I'm working with our tester to confirm. I have noticed DAP and focus issues. I will update issues as I have a better analysis.

**Comment by aradhikaIbm** [comment]
So we do see a partial issue. I've attached some images. 
modal_normal_zoom - shows what the entire dialog looks like 

For our tester, this what they see at 240 and 300:
Modal_240_zoom
Modal_300_zoom
FF: 60.2.1 esr

However, when I view it i do see a scroll. 
Modal_300_zoom_with_scroll
Chrome: Version 69.0.3497.100 (Official Build) (64-bit)

![modal_240_zoom](https://user-images.githubusercontent.com/22798731/46550794-13f7d180-c8a4-11e8-8013-5c468a97c8d0.jpg)
![modal_300_zoom](https://user-images.githubusercontent.com/22798731/46550795-13f7d180-c8a4-11e8-93e7-1968737ad42d.jpg)
![modal_300_zoom_with_scroll](https://user-images.githubusercontent.com/22798731/46550796-13f7d180-c8a4-11e8-93a4-684addf840b8.jpg)
![modal_normal_zoom](https://user-images.githubusercontent.com/22798731/46550797-13f7d180-c8a4-11e8-8db7-7315545d5abd.jpg)




FF: 62.0.2 (64-bit)

**Comment by tw15egan** [comment]
Closing this, as this does not seem to be an error with the component itself. Any custom overrides at 200% and 300% zoom can be done on the user end

**Comment by aradhikaIbm** [comment]
So we are not doing any custom overrides. The tester is is just zooming to 200% and 300% per the accessibility testing requirement. When a widget is accessible we need to also cover zoom and large text.

**Comment by carmacleod** [comment]
@aradhikaIbm 
The failing screen snapshots are at 240% and 300%.
Is the dialog ok at 200% zoom?
The requirement is only for 200%: https://www.ibm.com/able/guidelines/ci162/resize_text.html

@tw15egan 
Would a custom override allow dialogs with more content to zoom up to 200% reliably?
If so, maybe the Carbon doc for Modal could show an example of the override?
